common/build/eslint-config-fluid/src/test/enforce-no-restricted-tags-imports.test.js

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const { RuleTester } = require("eslint");
+const noRestrictedTagsImports = require("../custom-rules/no-restricted-tags-imports");
+
+const ruleTester = new RuleTester({
+	parser: require.resolve("@typescript-eslint/parser"),
+});
+
+const validCodeWithJSDoc = `
+/**
+ * @public
+ */
+export function publicFunction() {}
+
+import { publicFunction } from "./someModule";
+`;
+
+const validCode2WithJSDoc = `
+/**
+ * @internal
+ */
+export function internalExceptionFunction() {}
+
+import { internalExceptionFunction } from "./foo";
+`;
+
+const invalidCodeWithJSDoc = `
+/**
+ * @internal
+ */
+export function internalFunction() {}
+
+import { internalFunction } from "./internalModule";
+`;
+
+ruleTester.run("no-restricted-tags-imports", noRestrictedTagsImports, {
+	// Checks cases that should pass
+	valid: [
+		{
+			code: validCodeWithJSDoc,
+			// Valid import
+			options: [
+				{
+					tags: ["@internal", "@alpha"],
+				},
+			],
+		},
+		{
+			code: validCode2WithJSDoc,
+			// Restricted tag imported from exception
+			options: [
+				{
+					tags: ["@internal", "@alpha"],
+					exceptions: ["./foo"],
+				},
+			],
+		},
+	],
+	// Checks cases that should not pass
+	invalid: [
+		{
+			code: invalidCodeWithJSDoc,
+			// Import with restricted tag
+			options: [
+				{
+					tags: ["@internal", "@alpha"],
+					exceptions: ["./foo", "bar"],
```

Comment:
The exceptions don't seem to be relevant for this test. The simpler it is, the better :).
```suggestion
```